src/index.ts

```diff
@@ -221,11 +222,15 @@ export class GoogleToken {
 
     const signedJWT = jws.sign(toSign);
 
-    return axios
-        .post(GOOGLE_TOKEN_URL, {
-          grant_type: 'urn:ietf:params:oauth:grant-type:jwt-bearer',
-          assertion: signedJWT
-        })
+    return axios({
+             method: 'post',
+             url: GOOGLE_TOKEN_URL,
+             data: querystring.stringify({
```

Comment:
Curious. Why is `querystring.stringify()` needed here while it is not in the similar code in the test file?

Reply:
That's a great question!  I suspect nock is being "helpful" by automatically url decoding the content of the body.  My original test code was doing the `querystring.parse`, but it was failing the disableNetConnect because it didn't match bodies.